internal/setup/setup_test.go

```
@@ -822,7 +799,7 @@ var setupTests = []setupTest{{
 		},
 	},
 }, {
-	summary: "Archives with keyrings",
+	summary: "Archives with public keys",
```

We need a test with no public keys and we need to raise an error if that happens. We expect the archive to provide at least a public key in its YAML definition.

---

The tests will all need to contain public keys except the one mentioned above.

---

Okay, seems like a good idea.